src/crewai/agents/executor.py

```diff
@@ -185,6 +197,52 @@ def _iter_next_step(
             yield AgentStep(action=output, observation=observation)
             return
 
+        except BadRequestError as e:
+            print("Bad Request Error", e)
```

Comment:
_Suggestion_: Instead of print I think we should focus on using the Logger or the logging lib.